From ace481c3c6728ac5cf2439c0366e395a33337e6c Mon Sep 17 00:00:00 2001
From: Dylan Yip <dylan.yip@xilinx.com>
Date: Thu, 18 Mar 2021 17:03:00 -0700
Subject: [PATCH 1785/1851] arm64: zynqmp: Use new compatible strings for DP
 snd pcm

commit aee91708a188fb682de00666ac15aa4bfb5c88ac from
https://github.com/Xilinx/linux-xlnx.git

To remove a stack trace when removing and reprobing the DP snd pcm,
use the new DP pcm compatible strings to allow the driver to create
standardized names for each DP snd pcm.

Signed-off-by: Dylan Yip <dylan.yip@xilinx.com>
Acked-by: Varunkumar Allagadapa <varunkumar.allagadapa@xilinx.com>
Acked-by: Jianqiang Chen <jianqiang.chen@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 arch/arm64/boot/dts/xilinx/zynqmp.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/xilinx/zynqmp.dtsi b/arch/arm64/boot/dts/xilinx/zynqmp.dtsi
index b7a91769fb2c..17d25e4bfdb8 100644
--- a/arch/arm64/boot/dts/xilinx/zynqmp.dtsi
+++ b/arch/arm64/boot/dts/xilinx/zynqmp.dtsi
@@ -1177,13 +1177,13 @@
 			};
 
 			zynqmp_dp_snd_pcm0: zynqmp_dp_snd_pcm0 {
-				compatible = "xlnx,dp-snd-pcm";
+				compatible = "xlnx,dp-snd-pcm0";
 				dmas = <&xlnx_dpdma 4>;
 				dma-names = "tx";
 			};
 
 			zynqmp_dp_snd_pcm1: zynqmp_dp_snd_pcm1 {
-				compatible = "xlnx,dp-snd-pcm";
+				compatible = "xlnx,dp-snd-pcm1";
 				dmas = <&xlnx_dpdma 5>;
 				dma-names = "tx";
 			};
-- 
2.31.1

